[elasticsearch] update to 6.8.3

ryancragun opened the pull request:
* Update jvm opts to reflect defaults in the 6.8.3 distribution
* Fix deprecated console layout
* Bump to the latest stable 6.x version

Signed-off-by: Ryan Cragun <ryan@chef.io>

predominant commented:
### Testing

```
hab pkg build elasticsearch
source results/last_build.env
hab studio run "./elasticsearch/tests/test.sh ${pkg_ident}"
```

### Actual output

```
 ✗ Version matches
   (in test file elasticsearch/tests/test.bats, line 2)
     `expected_version=$(cut -d/ -f3 $TEST_PKG_IDENT)' failed
   cut: rakops/elasticsearch/6.8.3/20191001005008: No such file or directory
 ✓ Service is running
 ✓ Open port 9200
 ✓ Open port 9300
 ✓ Service Healthcheck

5 tests, 1 failure
```

@ryancragun Looks like there is an error in the version checking test. Can you confirm?

ryancragun commented:
@predominant Nice catch, the version of OpenJDK that it's built against puts a deprecation warning for the CMS garbage collector which breaks the version check test. I suspect these tests haven't worked since it was switched to OpenJDK11. I fixed the broken test and fixed shellcheck errors for the rest of them.

predominant commented:
Squash those commits, and we're good to go @ryancragun !

ryancragun commented:
@predominant sorry for taking so long, I was out for a little bit. :vulcan_salute:
